android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -82,6 +80,21 @@ public String getName() {
         return "MusicControlManager";
     }
 
+    private LruCache<String, Bitmap> mMemoryCache = new LruCache<String, Bitmap>(
```

Comment:
Please, don't use LruCache, use OkHttp's cache instead, as it takes advantage of file caching and it will share the same cache with React Native images.